docs/frontend/introduction-to-less.md

```diff
@@ -0,0 +1,114 @@
+# Indtroduction to LESS
+This document serves for describing specific behavior of CSS pre-processors LESS.
+
+## Import files
+In Shopsys framework we implement LESS by dividing styles into many LESS components. Every component has its own file. File name is same as component name. Getting all these components into one CSS file we accomplish by `@import` command.
+
+In example bellow we can see how looks syntax of this command:
+
+```css
+@import 'path/to/directory/component-filename.less';
+```
+
+#### Using @import
+Best way to import all related files is create one file, for example named as `main.less`. This file will contain only `@import` commands. Keep in mind where do you place this file. Imported path depends on where this file is placed.
+
+###### Unexcepted behavior
+There is one thing which you should keep in mind. When you try to import file which does not exist in given file path, compiler will try to find missing file in root directories of files, where is used `@import`.
+
+Let us show this at example. Assuming you have folder structure and files as is shown below:
+```
+│── root-main.less
+│── some-component.less
+└─── B
+    └── b-main.less
+```
+
+`B/b-main.less`:
+```css
+@import "some-component.less";
+```
+
+`some-component.less`:
+```css
+.some-component {
+    color: red;
+}
+```
+
+`root-main.less`:
+```css
+@import "B/b-main.less";
+```
+
+Result CSS of this example will be:
+```css
+.some-component {
+    color: red;
+}
+```
+As a explanation of this behavior, given in example above, is that compiler is trying to find file `some-component.less` firstly in folder where is placed `b-main.less`, then in directory of `root-main.less`. When it could not find required file in any directory, then it will thrown an error during compiling.
```

Comment:
```suggestion
As an explanation of this behavior, given in example above, is that compiler is trying to find a file `some-component.less` firstly in folder where is placed `b-main.less`, then in directory of `root-main.less`. When it could not find required file in any directory, then it will thrown an error during compiling.
```